Libraries/Components/Slider/Slider.js

```diff
@@ -28,6 +28,63 @@ type Event = Object;
 
 /**
  * A component used to select a single value from a range of values.
+ *
+ * * ### Usage
```

Comment:
Does this render correctly? While this is a docs only change, it's still a good idea to build and test that your change looks good.

Reply:
Hi @hramos, thank you for your feedback. It does render correctly. I tested it using Android. I am going to update the commit removing the extra `*`.

And I built `./website` locally, it looks good with the new changes. :)